packages/calcite-components/src/components/rating/rating.e2e.ts

```diff
@@ -810,21 +810,20 @@ describe("calcite-rating", () => {
           shadowSelector: `.${CSS.star}`,
           targetProp: "color",
         },
+        "--calcite-rating-color-press": {
+          shadowSelector: `.${CSS.star}`,
+          targetProp: "color",
+          state: { press: { attribute: "class", value: CSS.star } },
```

Comment:
The object argument to `state.press` is deprecated. Use a selector instead (see [example](https://github.com/Esri/calcite-design-system/blob/bbd57b1b7583134bc110736ba8221c768dd1bd10/packages/calcite-components/src/tests/commonTests/themed.ts#L51)).